baseapp/router.go

```diff
@@ -15,7 +15,7 @@ var _ sdk.Router = NewRouter()
 // NewRouter returns a reference to a new router.
 //
 // TODO: Either make the function private or make return type (router) public.
-func NewRouter() *router { // nolint: golint
+func NewRouter() *router { //nolint:golint
```

Comment:
```suggestion
func NewRouter() *router {
```

Reply:
Same here: exported function, unexported type

Reply:
Should I export `router` as well?

Reply:
yes, please